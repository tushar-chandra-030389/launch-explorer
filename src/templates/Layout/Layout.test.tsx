import { render } from "@testing-library/react";
import Layout from ".";

describe("Layout", () => {
  it("should render the layout wrapper", () => {
    const { asFragment } = render(<Layout />);
    const firstRender = asFragment();

    expect(firstRender).toMatchSnapshot();
  });
});
