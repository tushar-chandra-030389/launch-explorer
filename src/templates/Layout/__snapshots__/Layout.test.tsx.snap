// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render the layout wrapper 1`] = `
<DocumentFragment>
  <div
    class="h-screen grid grid-cols-12"
  >
    <div
      class="hidden sm:inline-grid	 sm:col-span-1 2xl:col-span-2"
    />
    <div
      class="col-span-12 sm:col-span-10 2xl:col-span-8 bg-background text-slate-50 h-full"
    />
    <div
      class="hidden sm:inline-grid sm:col-span-1 2xl:col-span-2"
    />
  </div>
</DocumentFragment>
`;
